Add option for extra powered by logo in footer (#3153)
ONSdigital/design-system: src/components/footer/_macro-options.md

| wide                 | boolean                        | false    | Set to “true” to increase the maximum width of the layout container to 1280px                                                                        |
| fullWidth            | boolean                        | false    | Set to “true” to increase the maximum width of the layout container to the full width of the viewport                                                |
| attributes           | object                         | false    | HTML attributes (for example, data attributes) to add to the footer                                                                                  |

> | logo                 | object`<LogoObject>`           | false    | Additional logo to be used in the footer.                                                                                                            |

Reviewer: ```suggestion
| logo                 | object`<LogoObject>`           | false    | Additional logo to go alongside the ONS logo in the footer                                                                                           |
```

> | extraPoweredByLogo   | object`<extraPoweredByLogo>`   | false    | Extra logo to go alongside the ONS logo in the footer                                                                                                |

Reviewer: We should move this line to be under the `poweredBy` param in the table because it makes sense for them to be next to eachother

| Name      | Type                                                        | Required | Description                 |
| --------- | ----------------------------------------------------------- | -------- | --------------------------- |
| itemsList | array`<ListItem>` [_(ref)_](/foundations/typography/#lists) | true     | A list of links for the row |

> ## LogoObject

Reviewer: ```suggestion
## Logo
```

| Name      | Type           | Required | Description                                                                             |
| --------- | -------------- | -------- | --------------------------------------------------------------------------------------- |

> | logoImage | HTML or string | false    | Any HTML to render an image for example embedded `<svg>` or `<img>`.                    |

Reviewer: "or string" can be removed from this, it doesn't accept a string. Also the description could do with updating the word "Any" can be removed

> ## Logo
> 
> | Name      | Type           | Required | Description                                                                                  |
> | --------- | -------------- | -------- | -------------------------------------------------------------------------------------------- |
> | logoImage | HTML or string | false    | Any HTML to render an image for example embedded `<svg>` or `<img>`.                         |
> | logoURL   | string         | false    | Wraps the additional logo in a link. Set the URL for the HTML `href` attribute for the link. |

Reviewer: ```suggestion
## ExtraPoweredByLogo

| Name      | Type           | Required | Description                                                                                  |
| --------- | -------------- | -------- | -------------------------------------------------------------------------------------------- |
| logoImage | HTML or string | false    | Any HTML to render an image for example embedded `<svg>` or `<img>`.                         |
| logoURL   | string         | false    | Wraps the extra logo in a link. Set the URL for the HTML `href` attribute for the link.      |
```
